en/flight_controller/snapdragon_flight_configuration.md

```diff
@@ -0,0 +1,62 @@
+# Configure Snapdragon
+## Autostart PX4 and Snap VIO
+In order to boot both the ROS node and PX4 automatically on bootup, edit the `/etc/rc.local` file on the snapdragon to look like this: (note that the first line changed too!)
+
+```
+#!/bin/bash -e
+#
+# rc.local
+#
+# This script is executed at the end of each multiuser runlevel.
+# Make sure that the script will "exit 0" on success or any other
+# value on error.
+#
+# In order to enable or disable this script just change the execution
+# bits.
+#
+# By default this script does nothing.
+
+# Generate the SSH keys if non-existent
+test -f /etc/ssh/ssh_host_dsa_key || dpkg-reconfigure openssh-server
+cd /home/linaro
+source /opt/ros/indigo/setup.bash
+source /home/linaro/ros_ws/devel/setup.bash
+sleep 5
+roslaunch snapdragon_mavros_vislam mavros_vislam.launch &
+sleep 10
+(cd /home/linaro && ./px4 ros_ws/src/snap-vislam-ros/px4_configs/ekf2/mainapp.co
+nf > mainapp_vislam.log)
+exit 0
+```
+
+## Put Snapdragon back into Access Point Mode
```

Comment:
Back? Did we put it out in the first place?

Reply:
We did... if you follow the tutorial linearly (Dev environment -> Snapdragon install -> configuration)
Added a comment explaining this

Reply:
That new note helps in that I was able to see that the mode change was "station mode". From that I was able to find the mention in the "Install ROS" section "Finally, set the system to station mode."

I am being quite pedantic, but you might change this new note as shown below:

> 


> If you followed the [installation process closely so far, your Snapdragon Flight will still be in station mode. This could be a problem if you want to fly it outdoors where your home Wi-Fi is no longer available, so we recommend putting it back into access point mode.

Reply:
Sure. I added a little explanation fluff in snapdragon_flight_software_installation.html#ros-install, "Set up your Snapdragon Flight to connect to your local Wi-Fi network so you can easily clone git repositories directly onto it." to make it clear.